Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM python:3.4-onbuild
+# https://github.com/docker-library/official-images/issues/2076
```

Comment:
What is this comment for?

Reply:
@jonnyshaw89 mostly to draw attention to the deprecation issue. Can be removed now that it's been discussed.